### ISSUE TITLE
Does IETNG communicate with any servers

### ISSUE THREAD

**Issue by queries123** [thundernest/import-export-tools-ng#397]
I just had a quick query... We're looking to audit several inboxes and export the emails into CSV format, but some may contain sensitive data. Are there any issues in terms of data protection / GDPR in relation to using the ImportExportToolsNG add on? I wasn't sure if this was covered in any documentation.

**Comment by cleidigh** [comment]
@queries123 
Thanks for the question.
I will be upfront about the limitations of my knowledge of GDPR, I need to educate myself.
I believe that part of the requirements have to do with protecting users data when it is stored online. I will answer in that context and you can follow up if I don't hit the mark.

First, IETNG does not interact with the user accounts, rather it works with the downloaded imap, pop or local messages. If you are exporting to csv and you have the right to view the original message data then exporting that data should not be an issue. Note this  is true for any type of export. Only the message data is exported in different formats

Hopefully this answers your question.
@cleidigh

**Comment by queries123** [comment]
That's great thanks - I wasn't sure if IETNG was sending the data out to a server somewhere to process it, in which case we'd have to look into the servers location in relation to GDPR etc. But if it instead does everything locally - there shouldn't then be any issue.

**Comment by cleidigh** [comment]
@queries123 
Yes everything is local. There is zero communication with any server.
@cleidigh

**Comment by queries123** [comment]
Perfect - thanks so much for clarifying :)
